src/Redux/Exchange/Exchange-test-utils.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+// As a basic setup, import your same slice reducers
+import userReducer from './Exchange';
+
+export default function renderWithProviders(
+  ui,
+  {
+    preloadedState = {},
+    // Automatically create a store instance if no store was passed in
+    store = configureStore({
+      reducer: { Exchange: userReducer },
+      preloadedState,
+    }),
+    ...renderOptions
+  } = {},
+) {
+  // eslint-disable-next-line
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃